### PR TITLE
Use gestures for good health.

### PR DESCRIPTION
**Pull request by cdata**
Paired w/ @azakus on this. Part 1 of 2 fixes for https://github.com/PolymerElements/paper-dialog/issues/39

/cc @ebidel @samccone

### CODE REVIEW

File: iron-overlay-behavior.html [hunk 1]
```diff
@@ -280,7 +280,7 @@
         this._toggleListener(this.opened, document, 'tap', this._boundOnCaptureClick, true);
         this._toggleListener(this.opened, document, 'keydown', this._boundOnCaptureKeydown, true);
         this._toggleListenersAsync = null;
-      });
+      }, 1);
```
Comment (samccone) [inline iron-overlay-behavior.html]: ha why do we need 1 here :)

**Comment by cdata** [comment]
Yikes, this breaks many tests. Please stand by!

File: iron-overlay-behavior.html (hunk 1 above)
Comment (cdata) [inline iron-overlay-behavior.html]: https://github.com/Polymer/polymer/issues/1305

Comment (samccone) [inline iron-overlay-behavior.html]: wooof ok :dog: :do_not_litter:

**Comment by azakus** [comment]
LGTM

**Pull request merged**